draft-ietf-quic-recovery.md

```diff
@@ -1282,15 +1309,28 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
+   InPersistentCongestion(congestion_period):
+     pto = smoothed_rtt + 4 * rttvar + max_ack_delay
+     return
+       congestion_period >
+         pto * (2 ^ kPersistentCongestionThreshold - 1)
```

Comment:
editorial nit: this looks a bit confusing because 'return' is on a line by itself. How about:

return congestion_period >
                    pto * (2 ^ kPersistentCongestionThreshold - 1)